src/features/jobs-moderation.ts

```diff
@@ -165,19 +165,26 @@ const jobModeration = async (bot: Client) => {
       if (cooldown.hasCooldown(message.author.id, "user.jobs")) return;
 
       cooldown.addCooldown(message.author.id, "user.jobs");
-      message.author
-        .send(`Your post to #job-board didn’t have any tags - please consider adding some of the following tags to the start of your message to make your offer easier to find (and to index correctly on https://reactiflux.com/jobs):
-
-[FOR HIRE] - you are looking for a job
-[HIRING] - you are looking to hire someone
-[INTERN] - this is an intern position, no experience required
-[REMOTE] - only remote work is possible
-[LOCAL] - only local work is possible (please remember to provide the country / city!)
-[VISA] - Your company will help with the visa process in case of successful hire
-
-Thank you :)
-
-:robot: This message was sent by a bot, please do not respond to it - in case of additional questions / issues, please contact one of our mods!`);
+      const thread = await message.startThread({ name: 'Warning' });
```

Comment:
This is more of a suggestion than a warning, since we use "warning" to mean moderator interventions elsewhere I don't want to confuse
```suggestion
      const thread = await message.startThread({ name: message.author.username });
```